graphql/types/ReturnRequest.graphql

```diff
@@ -121,7 +121,7 @@ type RefundPaymentData {
 type ReturnRequestItem {
   orderItemIndex: Int!
   quantity: Int!
-  condition: ItemCondition!
+  condition: ItemCondition
```

Comment:
Regarding the last part, I can refactor the option to be the inverse.
Instead of 'enabling' it, you 'disable' it, so if you don't have it, it's enabled by default.

Reply:
Since we are still on beta, I think we are fine and it's better to keep the pattern of enabling stuff. It was just for us to keep it in mind.